src/vm-repair/azext_vm_repair/_validators.py

```diff
@@ -0,0 +1,124 @@
+        output = _call_az_command(find_rescue_command)
+        rescue_list = json.loads(output)
+
+        # No rescue VM found
+        if not rescue_list:
+            raise CLIError('Rescue VM not found for {vm_name}. Please check if the rescue resources were removed.'.format(vm_name=namespace.vm_name))
+
+        # More than one rescue VM found
+        if len(rescue_list) > 1:
+            message = 'More than one rescue VM found:\n'
+            for vm in rescue_list:
+                message += vm['id'] + '\n'
+            message += '\nPlease specify the --rescue-vm-id to restore the disk-swap with.'
+            raise CLIError(message)
+
+        # One Rescue VM Found
+        namespace.rescue_vm_id = rescue_list[0]['id']
+        logger.info('Found rescue vm: %s', namespace.rescue_vm_id)
+
+    if not is_valid_resource_id(namespace.rescue_vm_id):
+        raise CLIError('Rescue resource id is not valid.')
+
+    rescue_vm_id = parse_resource_id(namespace.rescue_vm_id)
+    # Check if data disk exists on rescue VM
+    rescue_vm = get_vm(cmd, rescue_vm_id['resource_group'], rescue_vm_id['name'])
+    data_disks = rescue_vm.storage_profile.data_disks
+    if not data_disks:
+        raise CLIError('No data disks found on rescue VM: {}'.format(rescue_vm_id['name']))
+
+    # Populate disk name
+    if not namespace.disk_name:
+        namespace.disk_name = data_disks[0].name
+        logger.warning('Disk-name not given. Defaulting to the first data disk attached to the rescue VM: %s', data_disks[0].name)
+    else: # check disk name
+        if not [disk for disk in data_disks if disk.name == namespace.disk_name]:
+            raise CLIError('No data disks found on the rescue VM: \'{vm}\' with the disk name: \'{disk}\''.format(vm=rescue_vm_id['name'], disk=namespace.disk_name))
+
+def _prompt_rescue_username(namespace):
+
+    from knack.prompting import prompt, NoTTYException
+    try:
+        namespace.rescue_username = prompt('Rescue VM Admin username: ')
+    except NoTTYException:
+        raise CLIError('Please specify username in non-interactive mode.')
+
+def _prompt_rescue_password(namespace):
+    from knack.prompting import prompt_pass, NoTTYException
+    try:
+        namespace.rescue_password = prompt_pass('Rescue VM Admin Password: ', confirm=True)
+    except NoTTYException:
+        raise CLIError('Please specify password in non-interactive mode.')
+
+def _classic_vm_exists(cmd, resource_group_name, vm_name):
+    api_version = '2017-04-01'
+    classic_vm_provider = 'Microsoft.ClassicCompute'
+    vm_resource_type = 'virtualMachines'
+
+    resource_client = _resource_client_factory(cmd.cli_ctx).resources
+    vm = None
+    try:
+        vm = resource_client.get(resource_group_name, classic_vm_provider, '', vm_resource_type, vm_name, api_version)
+    except CloudError as cloudError:
+        # Resource does not exist or the API failed
+        logger.debug(cloudError)
+        return False
+    print('sd')
```

Comment:
What is this?

Reply:
oops, forgot to remove that after recent test. Thanks for catching that.